src/kemal/helpers/helpers.cr

```diff
@@ -149,14 +149,14 @@ private def multipart(file, env : HTTP::Server::Context)
   end
 
   if endb == 0
-    endb = fileb
+    endb = fileb - 1
```

Comment:
Why is this a must :open_mouth:

Reply:
When chrome asks for a video with a Range header, it sometimes misses off the second numebr. eg
```
Range: bytes=0-
```
When it does this, if you return the filesize in the `Content-Range` header it doesn't play the video. But if you subtract 1 it does. 
Other browsers always seem to pass a full range ie

```
Range: bytes=0-65535
```
So it doesn't matter.

The end already had -1 applied to it, but it was happening all the time, even when the browser specified the end, so I'm guessing that's why it broke in safari (and occasionally in firefox). I think was more a chrome bug than safari. I'll try and find info online to backup this theory.

Reply:
Line 179 has the comment `Range not satisfable, see 4.4 Note` This 200 could be changed to a 416 error. Not sure why it's just returning the whole file? Any idea what this note is referring to?

Reply:
Its referring to http://httpwg.org/specs/rfc7233.html#status.416. 
> Note: Because servers are free to ignore Range, many implementations will simply respond with the entire selected representation in a 200 (OK) response. That is partly because most clients are prepared to receive a 200 (OK) to complete the task (albeit less efficiently) and partly because clients might not stop making an invalid partial request until they have received a complete representation. Thus, clients cannot depend on receiving a 416 (Range Not Satisfiable) response even when it is most appropriate.

Reply:
Thank you @petoem :+1:

Reply:
@petoem Thanks...

Reply:
Updated. I'm testing with 
````
curl --range 0-1024 http://127.0.0.1:3000/downloads/mountain.mp4 -v
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connected to 127.0.0.1 (127.0.0.1) port 3000 (#0)
> GET /downloads/mountain.mp4 HTTP/1.1
> Host: 127.0.0.1:3000
> Range: bytes=0-1024
> User-Agent: curl/7.54.0
> Accept: */*
> 
< HTTP/1.1 206 Partial Content
< Connection: keep-alive
< X-Powered-By: Kemal
< Content-Type: video/mp4
< ETag: W/"1506587481"
< X-Content-Type-Options: nosniff
< Content-Length: 1025
< Accept-Ranges: bytes
< Content-Range: bytes 0-1024/8953605
````